ScheduleSetting/ScheduleTable/ScheduleTableCell/index.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {
   Button,
-  CloseIcon,
+  CloseSmallIcon,
```

Comment:
Thinking about this a bit more, why not use the `CloseIcon` with the `small` prop? 

Seems like having a `Close<Size>Icon` would lead to lots of icons